### PR TITLE
#137 use custom objectMapper for gateways

### LINKED ISSUE #137: Crawler gateways Json serialization is Camel_Case
**Current behavior**
Crawler gateways Json serialization is configured has default configuration: CAMEL_CASE

**Expected behavior**
serialization must be SNAKE_CASE

**Analysis**
Defined ObjectMapper in ``JacksonConfiguration.class`` is not the same as in the RestTemplate. Custom ObjectMapper has to be configured: https://stackoverflow.com/questions/9381665/how-can-we-configure-the-internal-jackson-mapper-when-using-resttemplate

### PR DESCRIPTION
**Pull request by maschleg**
closes #137

### CODE REVIEW

File: crawler/src/main/java/org/zalando/apidiscovery/crawler/ClientsConfiguration.java [hunk 1]
```diff
@@ -62,11 +67,14 @@ public RestTemplate oauth2Operations() {
...
 
     private RestTemplate buildOAuth2RestTemplate(final String tokenName) {
-        return new StupsOAuth2RestTemplate(new StupsTokensAccessTokenProvider(tokenName, accessTokens),
+        RestTemplate rt = new StupsOAuth2RestTemplate(new StupsTokensAccessTokenProvider(tokenName, accessTokens),
                 ClientHttpRequestFactorySelector.getRequestFactory(new TimeoutConfig.Builder()
                         .withReadTimeout(readTimeout)
                         .withConnectTimeout(connectTimeout)
                         .withConnectionRequestTimeout(connectionRequestTimeout)
                         .build()));
+        rt.getMessageConverters().add(0, new MappingJackson2HttpMessageConverter(objectMapper));
```
Comment (fmueller) [inline crawler/src/main/java/org/zalando/apidiscovery/crawler/ClientsConfiguration.java]: Just for sake of consistency, can you also add this to RestTemplate created in `oauth2Operations`?

Comment (maschleg) [inline crawler/src/main/java/org/zalando/apidiscovery/crawler/ClientsConfiguration.java]: ``oauth2Operations`` calls ``buildOAuth2RestTemplate`` ;)

Comment (fmueller) [inline crawler/src/main/java/org/zalando/apidiscovery/crawler/ClientsConfiguration.java]: Ups, ok. ^^

**Comment by fmueller** [comment]
👍

**Pull request merged**